src/js/components/Data/DataForm.js

```diff
@@ -262,15 +246,14 @@ export const DataForm = ({
   const contextValue = useMemo(() => ({ inDataForm: true }), []);
 
   const onSubmit = useCallback(
-    ({ value, touched }) => {
+    ({ value }) => {
       const nextValue = normalizeValue(value, formValue, views);
       resetPage(nextValue, formValue);
       setFormValue(nextValue);
-      if (onTouched) onTouched(transformTouched(touched, nextValue));
       onView(formValueToView(nextValue, views));
       if (onDone) onDone();
     },
-    [formValue, onDone, onTouched, onView, views],
+    [formValue, onDone, onView, views],
   );
 
   const onChange = useCallback(
```

Comment:
Do we still need `touched` in this function?

Reply:
Currently we do because touched is still used for the `debounceSearch` function in onChange.

Reply:
In that instance, `touched` is unrelated to `onTouched` and is coming directly from form.